e2e/tests/nightly/genomic-extraction-to-vcf.spec.ts

```diff
@@ -208,35 +212,67 @@ describe('Genomics Extraction Test', () => {
     // Not waiting for delete to finish.
   });
 
-  // Check creation status.
-  async function waitForComplete(page: Page, maxTime: number): Promise<boolean> {
+  async function waitForRuntimeComplete(page: Page, maxTime: number): Promise<boolean> {
```

Comment:
Why do we have so much custom code in this test case for waiting on a runtime to start? Isn't this a very central function which should be supported by a library or utility? Is this not duplicated somewhere else in the code?